src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/PrisonService.kt

```diff
@@ -72,7 +72,7 @@ class PrisonService(
   }
 
   fun getCurrentUserPrisonsList(): List<String> {
-    return prisonApiClient.getCurrentUserCaseLoads()?.filter { it.currentlyActive }?.map { caseLoad -> caseLoad.caseLoadId }
+    return prisonApiClient.getCurrentUserCaseLoads()?.map { caseLoad -> caseLoad.caseLoadId }
```

Comment:
can kick off a bulk comparison for an unselected prison but then not be able to view the comparison https://calculate-release-dates-dev.hmpps.service.justice.gov.uk/compare/list